src/main/java/edu/kit/ipd/crowdcontrol/objectservice/database/operations/TasksOperations.java

```diff
@@ -49,7 +49,7 @@ public boolean updateTask(TaskRecord taskRecord) throws IllegalArgumentException
      * @param experimentId the primary key of the experiment
      * @return the found task or empty if not found
      */
-    public Optional<TaskRecord> searchTask(String platform, int experimentId) {
+    public Optional<TaskRecord> findTask(String platform, int experimentId) {
```

Comment:
there was somewhere a cool blog entry about using search vs. find. The message of the entry was that you should only use find if you can assert that something will be found. according to that i would use search here, since the search can fail.